README.md

```diff
@@ -5,15 +5,27 @@ Implements the [TPCH benchmark](http://www.tpc.org/tpch/) for Postgres
 * The benchmark requires TPC-H dbgen:
 
 ```
-wget https://github.com/electrum/tpch-dbgen/archive/32f1c1b92d1664dba542e927d23d86ffa57aa253.zip
-unzip 32f1c1b92d1664dba542e927d23d86ffa57aa253.zip
-mv tpch-dbgen-32f1c1b92d1664dba542e927d23d86ffa57aa253 tpch-dbgen
-rm 32f1c1b92d1664dba542e927d23d86ffa57aa253.zip
+wget -q https://github.com/electrum/tpch-dbgen/archive/32f1c1b92d1664dba542e927d23d86ffa57aa253.zip -O tpch-dbgen.zip
+unzip tpch-dbgen.zip -d tpch-dbgen/ && rm tpch-dbgen.zip
```

Comment:
Did you test this? I think this might create two levels of directories?

Reply:
You are right. Let me fix this.